packages/graphql/src/schemaTypes/objectTypes/gql-Browser.ts

```diff
@@ -8,7 +8,7 @@ export const Browser = objectType({
   definition (t) {
     t.nonNull.string('channel')
     t.nonNull.boolean('disabled', {
-      resolve: () => false,
+      resolve: (source, args, ctx) => Boolean(!ctx.browser.isVersionSupported(source)),
```

Comment:
We have `disabled` and `isVersionSupported`, do we need both? I feel like we are missing some functionality like detecting browsers we know won't work (like safari), and in that case the disabled would make sense for browsers like that.

I think we should revert the change to `disabled` for now rather than having two properties that are the inverse of each other.

Reply:
oh, ok. i assumed that since `disabled` was already there and was already set to gray out those browsers in the list that it had some other meaning different than what we're doing here. hence why i left different behavior in the list for `disabled` vs `disabled && !isVersionSupported`. but i'd rather combine them if we won't lose any needed functionality.

Reply:
oh, maybe i misread? you're saying to keep both but put `disabled` back to always being false?

Reply:
I was guessing at the real meaning of `disabled` since it doesn't seem to be doing anything. Leaving it as it is requires less changes, but since it's mostly unused I'm ok with reducing it to one property for now if you think that's best.